src/turbopuffer.test.ts

```diff
@@ -467,7 +462,8 @@ test("sanity", async () => {
   );
 }, 10_000);
 
-test("connection_errors_are_wrapped", async () => {
+const t = isRuntimeFullyNodeCompatible ? it : it.skip;
+t("connection_errors_are_wrapped", async () => {
```

Comment:
what error do we get with the fetch client? I liked that previously it still tested it